server/plugin.go

```diff
@@ -173,9 +233,41 @@ func (p *Plugin) deleteEphemeralPost(userID, postID string) {
 }
 
 func (p *Plugin) updateJwtUserInfo(jwtToken string, user *model.User) (string, error) {
-	secret := p.getConfiguration().JitsiAppSecret
 	sanitizedUser := user.DeepCopy()
 
+	if p.getConfiguration().UseJaaS {
```

Comment:
Maybe pull this block out into its own function since we unconditionally return at the end of the block?